src/Casbin.AspNetCore/Transformers/BasicRequestTransformer.cs

```diff
@@ -0,0 +1,39 @@
+﻿
+
+using System;
+using System.Linq;
+using System.Security.Claims;
+using Casbin.AspNetCore.Abstractions;
+
+namespace Casbin.AspNetCore.Transformers
+{
+    public class BasicRequestTransformer : IRequestTransformer
+    {
+        public BasicRequestTransformer()
+        {
+            SubTransformer = context => context.User.FindFirst(ClaimTypes.Name).Value;
+            ObjTransformer = context => context.Data.ResourceName ?? string.Empty;
+            ActTransformer = context => context.Data.ActionName ?? string.Empty;
+        }
+
+        public BasicRequestTransformer(string? issuer) : this()
```

Comment:
This method is not test coveraged.

Reply:
I only implement the basic test now, I will add others later.